Gemfile

```diff
@@ -56,7 +56,7 @@ end
 
 gem 'bixby', '~> 5.0'
 gem 'blacklight', '~> 7.18', '< 7.25'
-gem 'blacklight-gallery', '~> 4.4.0'
+gem 'blacklight-gallery', '~> 4.1.0'
```

Comment:
Could we add a comment about what was bad when we tried to go higher than this?

Reply:
Turns out I can upgrade it it now, must have been me trying to get things to pass when there was that override that was necessary.